packages/venia-concept/templates/preloads.mst

```diff
@@ -1,3 +1,9 @@
 {{#urlResolver}}
 <script type="application/json" id="url-resolver">{ "type": "{{type}}", "id": "{{id}}" }</script>
 {{/urlResolver}}
+<script>
+    const fontStylesheet = document.createElement('link');
```

Comment:
As a link tag in the head, this font was a blocking resource. This does run the risk of a FOUC, but I'd rather have that than a blocking load.

Reply:
I went down a whole rabbit hole of [interesting research](https://codepen.io/tigt/post/async-css-without-javascript#the-future-5) on this but it does seem like this is the recommended way to accomplish async loading a stylesheet 👍 .